tests/sys/zuse/dawn.hoon

```diff
@@ -0,0 +1,282 @@
+/+  *test
+|%
+:: example hull for ~zod
+::
+++  hul
+  ^-  hull:constitution:ethe
+  :+  [0x0 0x0 0x0 0x0]
+    :*  ~
+        life=1
+         pass=2.448.360.348.730.164.860.814.441.775.703.143.856.915.192.920.
+        639.124.529.297.987.279.849.833.790.775.864.413.949.853.880.667.744.
+        188.597.545.066.664.466.963.044.328.182.155.965.137.512.758.548.384.
+        637.214.562
+        continuity-number=0
+        sponsor=[~ u=~zod]
+        escape=~
+    ==
+  [~ u=[spawn-proxy=0x0 spawn-count=2 spawned=~]]
+:: secret key for ~zod
+::
+++  sec
+  ^-  ring
+  0w8.Ugyke.eUOf2.NcHRo.tZA7r.P8vP6.DGKp4.yn-BI.etdzb.ucv75.WgRob.H1-7n.
+  4qCje.gc7z7.1i-3T.9~8UR.IGkGH.6NWR2
+:: Ships contract address
+::
+++  ships
+  %-  crip
+  %+  weld  "0x"
+  (render-hex-bytes:ethereum 20 `@`ships:contracts:constitution:ethe)
+:: this produces a 1000+ line payload, so we just check that it doesn't crash
+::
+++  test-give-czar
+  =/  zar  czar:give:dawn
+  ~!  zar
+  %+  expect-eq
+    !>  &
+    !>  ?=(^ zar)
+::
+++  test-give-hull
+  =/  oct
+    %-  as-octs:mimes:html
+    %+  rap  3
+    :~  '{"jsonrpc":"2.0","method":"eth_call","params":[{"data":"'
+        '0xb322069400000000000000000000000000000000000000000000000000000000'
+          '00000000'
```

Comment:
For calls like this and the ones in turf, it makes sense to put the function identifier (first 4 bytes) on a separate line from the first argument, so that you still end up with 32-byte lines for the arguments.

Sorry for being a pain about this, and thanks for the changes. The others look super good now!

Reply:
Thanks!